Yafc/Windows/ProjectPageSettingsPanel.cs

```diff
@@ -151,9 +153,9 @@ private class ExportRecipe {
 
             public ExportRecipe(RecipeRow row) {
                 Recipe = row.recipe.name;
-                Building = row.entity.name;
+                Building = row.entity?.name ?? "<No building selected>";
```

Comment:
Two of the bugs I remember fixing are this line and `row.fuel?.name`, two lines down. It is possible to arrange things so a recipe has no available crafting entities, which means that both row.entity and row.fuel will be null. Most of the code checks those for null, but these two lines didn't.